articles/libraries/lock/v10/index.md

```diff
@@ -120,10 +120,7 @@ document.getElementById('btn-login').addEventListener('click', function() {
 Embedding Lock within your application, rather than using the [Hosted Login Page](/hosted-pages/login), requires [cross-origin authentication](/cross-origin-authentication). In order to use embedded Lock via cross-origin authentication, you must do the following:
 
 * Set the [oidcconformant](/libraries/lock/v10/configuration#oidcconformant-boolean-) option to true
-* Set the [audience](/libraries/lock/v10/configuration#audience-string-) option
-* In the client settings area of the [Dashboard]($manage_url}), in the **Advanced Settings** menu, under the **OAuth** tab, turn on the **OIDC Conformant** and **Cross Origin Authentication** settings.
-
-    ![Cross-Origin Authentication switch](/media/articles/cross-origin-authentication/cross-origin-switch.png)
+* In the client settings area of the [Dashboard]($manage_url}), in the **Advanced Settings** menu, under the **OAuth** tab, turn on the **OIDC Conformant** setting.
 
```

Comment:
This is not necessary. COA works without OIDC Conformant flag in the server, but it needs it in the Lock options.